packages/contracts/build/contracts.json

```diff
@@ -0,0 +1,22 @@
+{
+    "Admin": "0x627306090abaB3A6e1400e9345bC60c78a8BEf57",
+    "Seller": "0xf17f52151EbEF6C7334FAD080c5704D77216b732",
+    "Buyer": "0xC5fdf4076b8F3A5357c5E395ab970B5B54098Fef",
+    "Arbitrator": "0x821aEa9a577a9b44299B9c15c88cf3087F3b5544",
+    "Affiliate": "0x0d1d4e623D10F9FBA5Db95830F7d3839406C6AF2",
+    "OGN": "0xF12b5dd4EAD5F743C6BaA640B0216200e89B60Da",
+    "DAI": "0x345cA3e014Aaf5dcA488057592ee47305D9B3e10",
+    "OKB": "0xf25186B5081Ff5cE73482AD761DB0eB0d25abfBF",
+    "USDT": "0x8f0483125FCb9aaAEFA9209D8E9d7b9C8B9Fb90F",
+    "Marketplace": "0xFB88dE099e13c3ED21F80a7a1E49f8CAEcF10df6",
```

Comment:
This file should be generated locally rather than checked in, as all the addresses refer to contracts on the local testnet

Reply:
I'm following the same pattern as in the DApp where those files are checked in but overwritten at startup. See https://github.com/OriginProtocol/origin/tree/master/packages/contracts/build

The challenge with removing those files is that the [contract deployer](https://github.com/OriginProtocol/origin/blob/master/packages/services/index.js#L184) is a graphql server and it can't get started without those files as otherwise it complains about missing imports! See this [line](https://github.com/OriginProtocol/origin/blob/master/packages/graphql/fixtures/populate-server.js#L15).

I'd suggest that for the short-term we go with this but as a separate PR it would be good to simplify contract deployment (especially since it is only 1 contract, the marketplace_v1) so that it does not depend on graphql anymore but just runs truffle.